python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 1.5.0
+
+### Feature:
+- Support Microsoft SQL as Database [#1538](https://github.com/astronomer/astro-sdk/pull/1538)
+- Support DuckDB as Database. `aql.merge` requires a fix on DuckDB as `UPDATE/IGNORE` clause is not supported as per [bug on duckdb](https://github.com/duckdb/duckdb/issues/6087) [#1695](https://github.com/astronomer/astro-sdk/pull/1695)
+- Add `load_options` param to accept `pandasOptions` in `load_file` operator [#1466](https://github.com/astronomer/astro-sdk/pull/1466)
+- Add `result_format` and `fail_on_empty` params to `run_raw_sql` operator [#1584](https://github.com/astronomer/astro-sdk/pull/1584)
+- Add support `validation_mode` as part of the `COPY INTO` command for snowflake. Specify the supported `validation_mode`; `RETURN_n_ROWS` or `RETURN_ERRORS` or `RETURN_ALL_ERRORS`. This instructs the `COPY INTO` command to validate the data files instead of loading them into the specified table; i.e. the `COPY INTO` command tests the files for errors but does not load them. [#1689](https://github.com/astronomer/astro-sdk/pull/1689)
```

Comment:
`Add support `validation_mode` as part of the `COPY INTO` command for the snowflake.` i think this should be sufficient.

Reply:
Should we not make it clear to our users what this change means? Like we did in past like https://github.com/astronomer/astro-sdk/blob/main/python-sdk/docs/CHANGELOG.md#features

Reply:
IMO, If someone is interested they can follow up on the ticket/PR.

Reply:
https://github.com/astronomer/astro-sdk/pull/1714/commits/4eb93d4e1542e69c9f763afefd5b62828860ed95